docs/collecting-data/collecting-from-own-applications/mobile-trackers/anonymous-tracking/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Anonymous Tracking"
+date: "2022-08-30"
+sidebar_position: 60
+---
+
+# Anonymous Tracking
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+(Available since v4)
```

Comment:
Does it make sense to make this more prominent, like
```
:::info

This feature available since v4

:::
```

Reply:
Changed it to the banner here and in a couple of other places.